client/src/components/Event/Event.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+
+export type UserRecord = {
+    name: string;
+    answer: "yes" | "no" | "if-needed";
+}
+
+export type DateRecord = {
+    timestamp: number;
+    records: UserRecord[];
+}
+
+// props komponenty Event
+export type EventProps = {
+    location?: string;
+    id: string;
+    title: string;
+    dates: DateRecord[];
+}
+
+export function Event({ location, title, dates }: EventProps) {
+    const users = Array.from(
+        new Set(dates.flatMap((d) => d.records.map((r) => r.name)))
+    );
+
+    return (
+        <div>
+            <h2>{title}</h2>
+            {location && <p>Místo konání akce: {location}</p>}
+
+            <table border={1}>
```

Comment:
V zadani to bylo zmineno, zvazit rozdeleni na vice komponent (jakkoliv je to treba tady "trochu moc", tak cilem je si to osvojit) - napr. udelat EventHeader komponentu, ktera zobrazi misto a lokaci :)